content/en/agent/kubernetes/control_plane.md

```diff
@@ -306,7 +308,28 @@ scheduler:
     bind-address: 0.0.0.0
 ```
 
+## Kubernetes on Amazon Elastic Kubernetes Service {#EKS}
+
+On EKS, AWS exposes API Server metrics via [a single endpoint][5]. This allows the Datadog Agent to obtain API Server metrics using endpoint checks as described in the [Kubernetes API Server metrics check documentation][6]. To conigure the check, add the following annotations to the `default/kubernetes` service:
+
+```yaml
+annotations:
+  ad.datadoghq.com/endpoints.check_names: '["kube_apiserver_metrics"]'
+  ad.datadoghq.com/endpoints.init_configs: '[{}]'
+  ad.datadoghq.com/endpoints.instances:
+    '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
+```
+
+Other control plane components are not exposed in EKS and cannot be monitored.
+
+## Kubernetes on Managed Services (Azure Kubernetes Service, Google Kubernetes Engine) {#ManagedServices}
+
+On most Managed Services the user cannot access the control plane components. As a result, it is not possible to run the `kube_apiserver`, `kube_controller_manager`, `kube_scheduler`, and `etcd` checks in these environments.
```

Comment:
```suggestion
On managed services, such as Azure Kubernetes Service (AKS) or Google Kubernetes Engine (GKE), the user cannot access the control plane components. As a result, it is not possible to run the `kube_apiserver`, `kube_controller_manager`, `kube_scheduler`, and `etcd` checks in these environments.
```

Reply:
I think it might be better to keep "most" since EKS is also a managed service; or should I use "other"?